pom.xml

```diff
@@ -60,6 +60,11 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>applicationinsights-web</artifactId>
+             <version>[1.0,)</version>

```

Comment:
Update version to 2.0.0-BETA for now and later to [2.0.,) once stable version is out.

Reply:
Done.